.github/workflows/test.yml

```diff
@@ -0,0 +1,64 @@
+name: Run linter and tests
+
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
```suggestion
      - main
      - master
```

Reply:
This prevents running the workflow twice when a push is made to a pull request's branch.